src/components/containers/ProfileSidebar/ProfileSidebar.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+// components
+import Avatar from '../../presentationals/Avatar/Avatar';
+import Button from '../../presentationals/Button/Button';
+
+// styles
+import './ProfileSidebar.scss';
+
+// config
+import { DEFAULT_IMAGE_URL } from '../../../utils/config';
+
+class ProfileSidebar extends Component {
```

Comment:
This can be turned to a stateless component.

Reply:
This has been made a stateless component and moved to the presentationals folder